pkg/api/pss_test.go

```diff
@@ -0,0 +1,171 @@
+package api_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/api"
+	"github.com/ethersphere/bee/pkg/logging"
+	"github.com/ethersphere/bee/pkg/pss"
+	resolverMock "github.com/ethersphere/bee/pkg/resolver/mock"
+	"github.com/ethersphere/bee/pkg/storage/mock"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/bee/pkg/trojan"
+	"github.com/gorilla/websocket"
+)
+
+func newTestWsServer(t *testing.T, o testServerOptions) *httptest.Server {
```

Comment:
This looks the same as the test server constructor, just returns the server. Nothing specific to ws, even more, there should not be a need for a specific ws test server. Existing newTestServer can just return also the server or ws client if needed.